### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Error getting cluster configurations with NEW HDInsight API

### ISSUE THREAD

**Issue by pamejujuy** [microsoft/azure-tools-for-java#3789]
IntelliJ build version: 2018.3.4 IC-183.5429.30
OS: Mac OS X
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.23.0-2018.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,22,1640x948,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Error getting cluster configurations with NEW HDInsight API. /subscriptions/486c672b-67fd-4bbc-afa8-94a2bac20f99/resourceGroups/marketMELI/providers/Microsoft.HDInsight/clusters/sparkmlb, throwable=com.microsoft.azure.hdinsight.sdk.cluster.HDInsightNewAPI.HDInsightNewApiUnavailableException: com.microsoft.azure.hdinsight.sdk.common.errorresponse.NotFoundHttpErrorStatus: {"error":{"code":"ResourceNotFound"

**Comment by wezhang** [comment]
```
Uncaught Exception Error getting cluster configurations with NEW HDInsight API. /subscriptions/486c672b-67fd-4bbc-afa8-94a2bac20f99/resourceGroups/marketMELI/providers/Microsoft.HDInsight/clusters/sparkmlb com.microsoft.azure.hdinsight.sdk.cluster.HDInsightNewAPI.HDInsightNewApiUnavailableException: com.microsoft.azure.hdinsight.sdk.common.errorresponse.NotFoundHttpErrorStatus: {"error":{"code":"ResourceNotFound","message":"The Resource 'Microsoft.HDInsight/clusters/sparkmlb' under resource group 'marketMELI' was not found."}}
```

```json
 {
  "error":
    {
      "code":"ResourceNotFound",
      "message":"The Resource 'Microsoft.HDInsight/clusters/sparkmlb' under resource group 'marketMELI' was not found."
    }
}
```

**Comment by t-rufang** [comment]
Actually this bug has already been fixed in #3638 long time ago.
@pamejujuy Please upgrade your IntelliJ to [the latest version](https://www.jetbrains.com/idea/download/#section=windows) first. Then upgrade the toolkit to the latest version. I will close this issue for now. If the error remains the same after you upgrade both IntelliJ and the toolkit, please reopen this issue again.
